lib/cargo.js

```diff
@@ -23,12 +30,19 @@ function provideBuilder() {
 
     settings: function (path) {
       var cargoPath = atom.config.get('build-cargo.cargoPath');
+      var with_verbose = function (name) {
+        if (atom.config.get('build-cargo.verbose')) {
+          return [name, '--verbose'];
+        } else {
+          return [name];
+        }
+      };
 
```

Comment:
ah wonderful. I have never programmed javascript before. Array concatenation is much better!